README.md

```diff
@@ -1,21 +1,94 @@
 # GitHub Classroom
 [![Build Status](https://travis-ci.org/education/classroom.svg?branch=master)](https://travis-ci.org/education/classroom) [![Code Climate](https://codeclimate.com/github/education/classroom/badges/gpa.svg)](https://codeclimate.com/github/education/classroom) [![Dependency Status](https://dependencyci.com/github/education/classroom/badge)](https://dependencyci.com/github/education/classroom)
 
-GitHub Classroom is a [Ruby on Rails](http://rubyonrails.org/) application designed to automate repository creation and access control, making it easy for teachers to distribute starter code and collect assignments on GitHub
 
-![GitHub Classroom screenshot](https://cloud.githubusercontent.com/assets/1311594/14748352/32f677b0-0887-11e6-9ac2-8aa06e4341fa.png)
-
-## How it works
-
-Assignments are the core of GitHub Classroom. Teachers can easily create an assignment and distribute it to students using a private invitation URL. Optional starter code can be provided for individual or group work. It's even possible to delegate assignment creation and management to co-teachers and teaching assistants by adding them as organization administrators.
-
-## Hacking on GitHub Classroom
+## The workflow you use as a developer, scaled for the needs of students. 
+
+Developers rarely work all by themselves, on a deadline, or ship something they’ll only use once (with no idea whether it actually works). 
+ 
+Wouldn’t students be better served by showing versions of their work, iterating, checking in on milestones and showing off the final product? 
+ 
+With GitHub Classroom you can set up the industry-standard workflow and free up your time to focus on teaching. 
+ 
+*Classroom is a teacher-facing tool that uses the GitHub API to enable the GitHub workflow for education.* 
+ 
+You create an Assignment with starter code and directions, send along one link, and students get their own “sandbox” copy of the repo to get started.
+ 
+Set due dates, track assignments in your teacher dashboard, or integrate other tools like testing frameworks. 
```

Comment:
Should we say this before it's merged? I'm hoping this'll be done this week though, so not a big deal.